source/backend/api/Services/AcquisitionFileService.cs

```diff
@@ -87,6 +87,42 @@ public Paged<PimsAcquisitionFile> GetPage(AcquisitionFilter filter)
+                                {
+                                    FileNumber = fileProperty.file.FileNumber ?? string.Empty,
+                                    LegacyFileNumber = fileProperty.file.LegacyFileNumber ?? string.Empty,
+                                    FileName = fileProperty.file.FileName ?? string.Empty,
+                                    MotiRegion = fileProperty.file.RegionCodeNavigation?.Description ?? string.Empty,
+                                    MinistryProject = fileProperty.file.Project != null ? $"{fileProperty.file.Project.Code} {fileProperty.file.Project.Description}" : string.Empty,
+                                    CivicAddress = (fileProperty.fp?.Property != null && fileProperty.fp.Property.Address != null) ? fileProperty.fp.Property.Address.FormatFullAddressString() : string.Empty,
+                                    GeneralLocation = (fileProperty.fp?.Property != null) ? fileProperty.fp.Property.GeneralLocation : string.Empty,
+                                    Pid = fileProperty.fp is not null && fileProperty.fp.Property.Pid.HasValue ? fileProperty.fp.Property.Pid.ToString() : string.Empty,
+                                    Pin = fileProperty.fp is not null && fileProperty.fp.Property.Pin.HasValue ? fileProperty.fp.Property.Pin.ToString() : string.Empty,
+                                    AcquisitionFileStatusTypeCode = fileProperty.file.AcquisitionFileStatusTypeCodeNavigation.Description,
+                                    FileFunding = fileProperty.file.AcquisitionFundingTypeCodeNavigation != null ? fileProperty.file.AcquisitionFundingTypeCodeNavigation.Description : string.Empty,
+                                    FileAssignedDate = fileProperty.file.AssignedDate.HasValue ? fileProperty.file.AssignedDate.Value.ToString("dd-MMM-yyyy") : string.Empty,
+                                    FileDeliveryDate = fileProperty.file.DeliveryDate.HasValue ? fileProperty.file.DeliveryDate.Value.ToString("dd-MMM-yyyy") : string.Empty,
+                                    FileAcquisitionCompleted = fileProperty.file.CompletionDate.HasValue ? fileProperty.file.CompletionDate.Value.ToString("dd-MMM-yyyy") : string.Empty,
+                                    FilePhysicalStatus = fileProperty.file.AcqPhysFileStatusTypeCodeNavigation != null ? fileProperty.file.AcqPhysFileStatusTypeCodeNavigation.Description : string.Empty,
+                                    FileAcquisitionType = fileProperty.file.AcquisitionTypeCodeNavigation != null ? fileProperty.file.AcquisitionTypeCodeNavigation.Description : string.Empty,
+                                    FileAcquisitionTeam = string.Join(", ", fileProperty.file.PimsAcquisitionFilePeople.Select(x => x.Person.GetFullName(true))),
+                                    FileAcquisitionOwners = string.Join(", ", fileProperty.file.PimsAcquisitionOwners.Select(x => x.FormatOwnerName())),
```

Comment:
I think there's potential for NPE errors by not checking for nulls on a few properties here:

- fileProperty.file.AcquisitionFileStatusTypeCodeNavigation.Description
- fileProperty.file.PimsAcquisitionFilePeople.Select(...)
- fileProperty.file.PimsAcquisitionOwners.Select(...)

Reply:
Added check for status. each item is checked and collection are initialized as empty list from the parent constructor.